### PR TITLE
fixes #290 - Enhanced review script to cover new fields and added size constraint

### LINKED ISSUE #290: Enhance review script to include newly added fields
Enhance review script to cover newly added fields in store listing page.

### PR DESCRIPTION
**Pull request by djmgit**
Fixes issue #290, enhanced review script to cover newly
added fields and added size constraint to fields

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
error2

**Comment by djmgit** [comment]
@Achint08 @kavithaenair @singhpratyush @SKrPl please review.

File: docs/tutorial.md [hunk 1]
```diff
@@ -242,6 +265,26 @@ Proper app documentation as mentioned above so that your app can be featured on
 
 **A properly configured app.json in the root directory of your app**
 
+## Making sure your app is ready for getting published
+
+Before publishing your app (as mentioned below) you must make sure your app is ready for being
+published. You can either do this by going through the above guidelines and cross checking whether
+your app statisfies all the point or not or else you can use review tool to automatically review
```
Comment (kavithaenair) [inline docs/tutorial.md]: Typo

**Comment by kavithaenair** [comment]
@singhpratyush , if the length of the string exceeds, it may look something like this http://apps.loklak.org/details.html?q=emojiHeatmapper. So @djmgit must have taken that into consideration.

**Comment by djmgit** [comment]
@vibhcool exactly,if the lengths become too long, then the right section will go below the left section, there by distorting the design of the page.So I have added a length constraint.

**Comment by vibhcool** [comment]
rebase please :)

File: docs/tutorial.md (hunk 1 above)
Comment (kavithaenair) [inline docs/tutorial.md]: @djmgit it must be `satisfies`.

**Comment by djmgit** [comment]
@kavithaenair fixed typo

@vibhcool rebased

**Pull request merged**